rosbag2_py/test/common.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Why is this change needed? (same comment for other test files)

Reply:
It isn't needed, removed in https://github.com/ros2/rosbag2/pull/531/commits/9b14e40f90c44c41d3565e592467823be8a0c251.